<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="txtSearch"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="resultsTab">
       <attribute name="title">
        <string>Search results</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="treeResultsList">
          <column>
           <property name="text">
            <string>Filename</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last modified</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="previewsTab">
       <attribute name="title">
        <string>Previews</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1185</width>
             <height>384</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblScale">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboScale">
            <property name="currentText">
             <string>25%</string>
            </property>
            <item>
             <property name="text">
              <string>25%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>75%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>150%</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblPdfPreviewPage">
            <property name="text">
             <string>Page:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPreviewPage">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblTotalPreviewPagesCount">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="indexerTab">
       <attribute name="title">
        <string>Index</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBoxPaths">
          <property name="title">
           <string>Add paths to scan</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLineEdit" name="txtPathScanAdd"/>
           </item>
           <item row="3" column="0" colspan="5">
            <widget class="QListWidget" name="lstPaths"/>
           </item>
           <item row="1" column="3">
            <widget class="QToolButton" name="btnDeletePath">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnChoosePath">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btnAddPath">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ignore patterns, separated by ';'. Example: *.js;*Downloads*</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QGroupBox" name="groupBoxIndexProgress">
          <property name="contextMenuPolicy">
           <enum>Qt::PreventContextMenu</enum>
          </property>
          <property name="title">
           <string>Index Progress</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="lblPathsFound">
               <property name="text">
                <string>Paths found:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblPathsFoundValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lblAdded">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Added:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblAddedValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lblSkipped">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Skipped:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblSkippedValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="lblFailed">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Failed:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblFailedValue">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLineEdit" name="txtIgnorePatterns"/>
        </item>
        <item row="10" column="0">
         <widget class="QPushButton" name="btnStartIndexing">
          <property name="text">
           <string>Start indexing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblSearchResults">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="previewProcessBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulooqs">
    <property name="title">
     <string>looqs</string>
    </property>
    <addaction name="menuOpenConfigInTextEditorAction"/>
    <addaction name="menuAboutAction"/>
    <addaction name="menuAboutQtAction"/>
   </widget>
   <addaction name="menulooqs"/>
  </widget>
  <action name="menuAboutAction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menuOpenConfigInTextEditorAction">
   <property name="text">
    <string>Open config in text editor</string>
   </property>
  </action>
  <action name="menuAboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
